websites/T/Ticketer/presence.ts

```diff
@@ -0,0 +1,63 @@
+const presence = new Presence({
+		clientId: "1003750528242692096",
+	}),
+	browsingUnix = Math.floor((Date.now() / 1000) | 0);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "https://i.imgur.com/XDt1aUu.png",
+		},
+		{ pathname } = document.location;
+	let title: HTMLElement, logo: HTMLElement, dashboardInfo: HTMLElement;
+	presenceData.startTimestamp = browsingUnix;
+	switch (pathname) {
+		case "/commands": {
+			presenceData.details = "Viewing the commands-page";
+			break;
+		}
+		case "/changelog": {
+			presenceData.details = "Viewing the changelog-page";
+			break;
+		}
+		case "/contribute": {
+			presenceData.details = "Viewing the contribution-page";
+			break;
+		}
+		case "/dashboard": {
+			presenceData.details = "Browsing the dashboard";
```

Comment:
Would be better if those case are in a Record like here : https://github.com/PreMiD/Presences/blob/main/websites/V/Voiranime/presence.ts

Reply:
I kine like the way it currently is easy to read. That just seems like an extra step.

Reply:
I tried adding Records like you but they would require way to much code change and I think its easier if I stick to my simple solution!